### PR TITLE
add plugin manager info to README.md

### PR DESCRIPTION
**Pull request by ernierasta**
Just info about automatic install, which should be preferred IMO.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,23 @@ The files in this repository provide support for syntax highlighting in Vim.
 They must be installed in your vim runtimepath to work.
 
 ## Instructions
+
+### Automated install
```
Comment (ollykel) [inline README.md]: I'd prefer to title this section "Using a Plugin Manager"

File: README.md [hunk 2]
```diff
@@ -3,6 +3,23 @@ The files in this repository provide support for syntax highlighting in Vim.
...
+Plug 'ollykel/v-vim'
+
+```
+Reload config (or save & exit vim/neovim). Than:
+```vim
+:PlugInstall
+```
+Enjoy!
+
+### Manual way
```
Comment (ollykel) [inline README.md]: Prefer to title this "Using install.sh or Manual Installation"

**Review by ollykel — CHANGES_REQUESTED** [review]
Thanks for this one too! Just change a couple headers and it'll be good to go.

File: README.md (hunk 1 above)
Comment (ernierasta) [reply README.md]: Yeah, that is better.

File: README.md (hunk 2 above)
Comment (ernierasta) [reply README.md]: done

**Review by ollykel — APPROVED** [review]
Perfect, thank you!

**Review by Franklin257 — APPROVED** [review]

**Pull request merged**